### ISSUE TITLE
It would be nice, if you cold click on an entry in the TODO list...

### ISSUE THREAD

**Issue by benediktr** [jonathandelgado/SublimeTodoReview#136]
... and this will take you directly to the position.

I expect that, because the Sublime `Find` functionality works the same way.

Best regards,
Benedikt

**Comment by svenhofstede** [comment]
@benediktr You can use the arrow keys to the navigate the list of entries. Hitting Enter will then take you straight to the location within the code. 

I hope I understood your request correctly :)

**Comment by benediktr** [comment]
Okay, thank you for the feedback!

It's useful, one you know it :-) :-)

**Comment by FichteFoll** [comment]
So, sublime text actually does have ways to interface with mouse events (i.e. converting x y coordinates to text points), but the problem is that mouse bindings don't have a context thing, meaning each mouse binding would be active on every view. This is especially problematic because you can't multi-bind the same mouse events, such as double-clicking.

There are different ways to hook into this however, including: 

1. on_window_command listener
2. doing the same thing as the "Find Results" panel, i.e. setting 'result_file_regex' and 'result_line_regex' on the view (see docs on output panels)
3. have the user insert the mouse-binding by himself, thus avoiding conflicts caused by packages.

I would appreciate if you could re-open this issue so that I could potentially see it more easily should I end up working on this package. ;)

**Comment by jonathandelgado** [comment]
@FichteFoll Yeah, excuse the brevity of my last comment on this issue. We have tried a slew of options; this package actually had mouse maps for a while, but I started getting a lot of issues with them conflicting with other things, for example, take a look at #78. I'll reopen that issue for you, since that is where a majority of the mouse discussion took place.

I would be interested in looking at the `Find Results` view, I hadn't considered it before, but I would assume off the top that it would override the current render method we are using.
